cmd/glasskube/cmd/describe.go

```diff
@@ -176,6 +202,31 @@ func printReferences(ctx context.Context, pkg *v1alpha1.Package, manifest *v1alp
 	}
 }
 
+func referencesAsMap(
+	ctx context.Context,
+	pkg *v1alpha1.Package,
+	manifest *v1alpha1.PackageManifest,
+) []map[string]string {
+	references := []map[string]string{}
+	for _, ref := range manifest.References {
+		reference := make(map[string]string)
+		reference["label"] = ref.Label
+		reference["url"] = ref.Url
+		references = append(references, reference)
+	}
+	if pkg != nil {
+		repo := cliutils.RepositoryClientset(ctx)
+		repoClient := repo.ForPackage(*pkg)
+		if url, err := repoClient.GetPackageManifestURL(manifest.Name, pkg.Spec.PackageInfo.Version); err == nil {
+			reference := make(map[string]string)
+			reference["Label"] = "Glasskube Package Manifest"
+			reference["URL"] = url
```

Comment:
Please start property names with lowercase characters when converting to JSON/YAML